src/target/riscv/riscv.c

```diff
@@ -4501,6 +4583,27 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Execute a sequence of 32-bit instructions using the program buffer. "
 			"The final ebreak instruction is added automatically, if needed."
 	},
+	{
+		.name = "equality_match_trigger_enable",
+		.handler = riscv_equality_match_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use equality match trigger in bp/wp"
+	},
+	{
+		.name = "napot_trigger_enable",
+		.handler = riscv_napot_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use NAPOT trigger in bp/wp"
+	},
+	{
+		.name = "ge_lt_trigger_enable",
```

Comment:
The existing commands follow the naming scheme `set_something` or `set_enable_something` but the new commands are named differently.

I am fine with any naming convention but would like this to be consistent for all RISC-V commands, and would like this to be accepted in upstream without the need to redo it.

@timsifive This is now a recurring topic. What is your opinion or preference?

Reply:
Addressed